docs/api-docs/store-management/shipping/shipper-hq.md

```diff
@@ -15,7 +15,7 @@ When you enable ShipperHQ on a store, additional fields become available on the
 
 These field values can be set in the Control Panel by the merchant as well as using one of the following:

```

Comment:
```suggestion
These field values can be set in the control panel by the merchant as well as using one of the following:

```

Reply:
Changed to lowercase per [BC Glossary](https://intranet.bigcommerce.com/display/SST/BigCommerce+Glossary#BigCommerceGlossary-C).